### ISSUE TITLE
apache-noscript does not match on 'Primary script unknown'

### ISSUE THREAD

**Issue by K2DLS** [fail2ban/fail2ban#2466]
The following errors are logged to the apache error_log:

```
[Tue Jul 09 14:27:42.650548 2019] [proxy_fcgi:error] [pid 22075:tid 140322524440320] [client 127.0.0.1] AH01071: Got error 'Primary script unknown'
[Tue Jul 09 14:27:43.921468 2019] [proxy_fcgi:error] [pid 22075:tid 140322507654912] [client 127.0.0.1] AH01071: Got error 'Primary script unknown'
[Tue Jul 09 14:27:45.825009 2019] [proxy_fcgi:error] [pid 22075:tid 140322403243776] [client 127.0.0.1] AH01071: Got error 'Primary script unknown'
[Tue Jul 09 14:27:47.737391 2019] [proxy_fcgi:error] [pid 22075:tid 140322499262208] [client 127.0.0.1] AH01071: Got error 'Primary script unknown'
[Tue Jul 09 14:38:47.195425 2019] [proxy_fcgi:error] [pid 22075:tid 140322361280256] [client 127.0.0.1] AH01071: Got error 'Primary script unknown'
[Tue Jul 09 14:38:54.368465 2019] [proxy_fcgi:error] [pid 22075:tid 140322403243776] [client 127.0.0.1] AH01071: Got error 'Primary script unknown'
```
Running fail2ban-regex /var/log/httpd/error_log /etc/fail2ban/filter.d/apache-noscript.conf provides 0 hits.  This error is supposed to be trapped.

filter.d/apache-noscript contains:
`failregex = ^(?:does not exist|not found or unable to stat): <script>\b
            ^'<script>\S*' not found or unable to stat
            ^error '[Pp]rimary script unknown\\n'`

If I remove the \\\\n from the line immediately above, fail2ban then matches.  I believe that the filter is in error.

**Comment by sebres** [comment]
Duplicate of #2378.

**Comment by K2DLS** [comment]
Not sure why closed as a dupe of #2378.  These errors appear in my error_log.  2378 seems to be about errors that used to be in the error log no longer being there.  One of us is missing something.

**Comment by sebres** [comment]
Hmm... looks like you're right.
Will fix it now.
Thanks!
